CrowsZA/functions/fn_surrenderChanceZeus.sqf

```diff
@@ -0,0 +1,58 @@
+/*/////////////////////////////////////////////////
+Author: Landric
+			   
+File: fn_surrenderChanceZeus.sqf
+Parameters: pos, unit
+Return: none
+
+*///////////////////////////////////////////////
+params [["_pos",[0,0,0],[[]],3], ["_unit",objNull,[objNull]]];
+
+private _onConfirm =
+{
+	params ["_dialogResult","_in"];
+	_in params [["_pos",[0,0,0],[[]],3], "_unit"];
+
+	private _units = if(isNull _unit) then {
+		private _u = [];
+		{ _u insert [-1, units _x, true]; } forEach ((ASLToAGL _pos) nearEntities [["Man"], (_dialogResult deleteAt 0)]);
```

Comment:
You should use ``CAManBase`` instead of ``Man``. As that is more specifically "humans". Animals also inherit from ``Man``. (I know... its super logical....)

Reply:
Ah hah, did not know that. Fixed!